Makefile

```diff
@@ -19,6 +19,12 @@ TEST_COMPILE_OUTPUT = build/_output/bin/unifiedpush-operator-test
 code/run: code/gen
 	operator-sdk up local
 
+.PHONY: code/debug
+code/debug: code/gen
+	export WATCH_NAMESPACE=unifiedpush
+	dlv --listen=:2345 --headless=true --api-version=2 exec ./build/_output/bin/unifiedpush-operator-local  -- 
```

Comment:
It will not work probably. Run the command then do an echo on the ENV VAR.
See here in MSS OPER how we solve it. https://github.com/aerogear/mobile-security-service-operator/blob/master/Makefile#L169

Reply:
It works; I've been using it.